os/include/tinyara/mm/mm.h

```diff
@@ -254,6 +249,11 @@ typedef size_t mmaddress_t;		/* 32 bit address space */
 #error Unknown CONFIG_ARCH option, malloc debug feature wont work.
 #endif
 
+/* typedef is used for defining size of address space */
+
+#ifdef CONFIG_DEBUG_MM_HEAPINFO
+typedef size_t mmaddress_t;             /* 32 bit address space */
```

Comment:
pls change commit message 
ARCH_GET_ADDRESS  ==> ARCH_GET_RET_ADDRESS

Reply:
Thanks for pointing it out. Modified.